docs/server/installation.rst

```diff
@@ -214,7 +214,7 @@ run it, just issue:
 
 .. code-block:: bash
 
-   (env) $ pootle runserver
+   (env) $ pootle runserver --insecure
```

Comment:
Needs a big red warning about running with ``--insecure`` just as is in the Django docs.  Or at least point to Django's concern.